src/app/index.jsx

```diff
@@ -91,11 +91,12 @@ export default function App() {
           <p>Follow these steps to put your device into QDL mode:</p>
           <ol>
             <li>Unplug the device and wait for the LED to switch off.</li>
-            <li>Connect the device to your computer using the <strong>lower</strong> <span className="whitespace-nowrap">USB-C</span> port.</li>
+            <li>Connect the device to your computer using the <strong>lower</strong> <span className="whitespace-nowrap">USB-C</span> port <strong>①</strong>.</li>
+            <li>If the LED blinks red, connect power to <span className="whitespace-nowrap">OBD-C</span> port <strong>②</strong>.</li>
```

Comment:
I don't think we should add a condition here if we don't have an explanation yet

```suggestion
            <li>Connect power to the <span className="whitespace-nowrap">OBD-C</span> port <strong>②</strong>.</li>
```